src/index.js

```diff
@@ -150,8 +155,101 @@ const renderLevel = (root, levelNumber) => {
         .join("")}
     </tr>
   `;
+
+  renderSettings = () => {  
+    const settingsHTML = `
+    <div id="modal">
+      <div class="modal-content">
+        <span class="close material-symbols-outlined">cancel</span>
+        <h1>Setting</h1>
+      </div>
+    </div>
+    `;
+    document.querySelector("#overlay-screen").innerHTML = settingsHTML;
+
+    document.querySelector(".close").addEventListener("click", () => {
+      document.querySelector("#modal").style.display = "none";
+    });
+  };
+
+  renderHelp = () => {  
+    const helpHTML = `
+    <div id="modal">
+      <div class="modal-content">
+        <span class="close material-symbols-outlined">cancel</span>
+        <h1>How To Play</h1>
+        <p>Objective: Push the boxes to all of the goals to complete the level in the least amount of moves.</p>
+        <p>Use the mouse or keyboard to move around the board.</p>
+        <p>Keyboard Shortcuts:</br>
+          direction: arrow keys;  undo: z;  reset: r
+        </p>
+      </div>
+    </div>
+    `;
+    document.querySelector("#overlay-screen").innerHTML = helpHTML;
+
+    document.querySelector(".close").addEventListener("click", () => {
+      document.querySelector("#modal").style.display = "none";
```

Comment:
See above comment. Better to remove than hide I think, since we're not planning on showing it again.